index.html

```diff
@@ -841,6 +841,65 @@ <h3 class="r-fit-text" style="margin-top: 8rem;">
                     </section>
                 </section>
 
+                <!-- Honourable mention -->
+                <section>
+                    <section data-background-image="images/TyPower.svg"
+                             data-background-size="15%" data-background-position="right">
+                        <h2>Please also have a look at the following teams</h2>
```

Comment:
To be consistent with the existing code, please put the opening and closing `h2` tags on separate lines.

Reply:
Set h2 tag on separate lines.